src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionPropertyIndex.cs

```diff
@@ -77,7 +77,7 @@ public MotionPropertyIndex(MotionPropertyIndex motionPropertyIndex) : this(Inter
         /// Assignment operator.
         /// </summary>
         /// <param name="motionPropertyIndex">Source object to be assigned.</param>
-        /// <returns>Reference to this.</returns>
+        /// <since_tizen> 11 </since_tizen>
```

Comment:
Please don't remove `<return>`. This is internal - not a part of API, so it will not be visible nor accessible.

Reply:
It was my mistake. Assign API is not public open. will remove.